src/main/java/pl/touk/sputnik/review/Review.java

```diff
@@ -80,7 +81,9 @@ public void addError(String source, Violation violation) {
     }
 
     private void addError(@NotNull ReviewFile reviewFile, @NotNull String source, int line, @Nullable String message, Severity severity) {
-        reviewFile.getComments().add(new Comment(line, String.format(COMMENT_FORMAT, source, severity, message)));
+        reviewFile.getComments().add(new Comment(line, MessageFormat.format(
+                ConfigurationHolder.instance().getProperty(GeneralOption.COMMENT_FORMAT),
+                source, severity, message)));
```

Comment:
ReviewFormatter would be great. DTO names are bad, I just wanted to share my concept with container objects.